<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSelectSystem</class>
 <widget class="QDialog" name="DialogSelectSystem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>199</width>
    <height>205</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>国土基本図郭プラグイン</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>図葉</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="radioButton_2500">
        <property name="text">
         <string>2,500分の１</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_5000">
        <property name="text">
         <string>5,000分の１</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_2nd">
        <property name="text">
         <string>二次メッシュ</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>平面直角座標系</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="comboBox_system">
        <property name="currentIndex">
         <number>-1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>系</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogSelectSystem</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogSelectSystem</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_2nd</sender>
   <signal>clicked()</signal>
   <receiver>DialogSelectSystem</receiver>
   <slot>onCheck2nd()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_5000</sender>
   <signal>clicked()</signal>
   <receiver>DialogSelectSystem</receiver>
   <slot>onCheck5000()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_2500</sender>
   <signal>clicked()</signal>
   <receiver>DialogSelectSystem</receiver>
   <slot>onCheck2500()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>26</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onCheck2500()</slot>
  <slot>onCheck5000()</slot>
  <slot>onCheck2nd()</slot>
 </slots>
</ui>
